app/metrics.yaml

```
@@ -6072,7 +6088,8 @@ browser.search:
       - android-probes@mozilla.com
       - erichards@mozilla.com
     expires: never
-
+    no_lint:
+      - BASELINE_PING
```

```suggestion
      - BASELINE_PING

```

Let's restore the newline after this.